src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -311,125 +311,125 @@ class Power(
         6: "MW",
     },
 ):
-    """A power quantity."""
+    """A active_power quantity."""
```

Comment:
We'd have to rename the type to `ActivePower` as well,  because it has a different unit from reactive (VAR) and apparent (VA) powers.